fava/ext/portfolio_list/__init__.py

```diff
@@ -18,15 +18,11 @@ class PortfolioList(FavaExtensionBase):  # pragma: no cover
     """Sample Extension Report that just prints out an Portfolio List.
     """
 
-    def __init__(self, ledger):
-        super().__init__(ledger)
+    # pylint: disable=eval-used
+    def __init__(self, ledger, extension_config):
+        super().__init__(ledger, extension_config)
+        self.config = eval(extension_config, {}, {})
```

Comment:
Don't use `eval` and rather `ast.literal_eval`. Also I think we should do this for the extension

Reply:
Done - https://github.com/beancount/fava/pull/872/commits/32c4557ad79b5820e2a32bd8365025516f8f76bb

> Instead of the `active_ext` hack, I'd just pass the active extension to the template.
Done in https://github.com/beancount/fava/pull/872/commits/da80a9241f1080a1d1fd107f6ed156afd23a70df#diff-6d90976a6c500672baef2abdbc814f8f

> Please remove the `extensions-dir` option, if you want to use extensions from different paths, just use a python module...
Done, just need to squash.